pkg/scheduler/plugins/overcommit/overcommit.go

```diff
@@ -105,8 +105,14 @@ func (op *overcommitPlugin) OnSessionOpen(ssn *framework.Session) {
 			return util.Permit
 		}
 
-		//TODO: if allow 1 more job to be inqueue beyond overcommit-factor, large job may be inqueue and create pods
 		jobMinReq := api.NewResource(*job.PodGroup.Spec.MinResources)
+		jobQueueAllocatableResources := ssn.UnderAllocatableResources(ssn.Queues[job.Queue])
```

Comment:
`UnderAllocatableResources` is registered in another plugin. This will cause the dependency between plugins.